tox.ini

```diff
@@ -33,7 +33,9 @@ deps =
     djangomain: git+https://github.com/django/django.git@main#egg=Django
 
     wagtail5.2: wagtail>=5.2,<6.0
-    wagtail6.0: git+https://github.com/wagtail/wagtail.git@stable/6.0.x#egg=wagtail
+    wagtail6.0: wagtail>=6.0,<6.1
```

Comment:
Wagtail 6.0 is no longer supported, so may as well drop it from the matrix

Reply:
Ta, changed here: https://github.com/wagtail-nest/wagtail-modeladmin/pull/41/commits/b8f2779856f834d6d353a2d37a60a1ecd6f27d90